components/brave_shields/browser/brave_shields_util.cc

```diff
@@ -335,50 +406,121 @@ DomainBlockingType GetDomainBlockingType(HostContentSettingsMap* map,
   return DomainBlockingType::kNone;
 }
 
+// Since Shields work at the domain level, we create a pattern for
```

Comment:
shields does not work at the domain level, this changes behavior and needs to be fixed